maintainer/install_hole.sh

```diff
@@ -31,6 +31,7 @@
 set -o errexit -o nounset
 
 SCRIPT=$(basename $0)
+MAINTAINER=$(cd "$(dirname "$0")"; pwd -P)
```

Comment:
I'm going to leave these changes here for now in case we revert to using the pre-built binaries. We can clean up in a future PR.

Reply:
I think you can remove the `install_hole` script. Perhaps we should pin the download from our hole repo (as we're pullling from master) to a tag or commit so that we ensure we know which version of (customized) hole we're building. But otherwise I don't see us going back to the binaries.

The next thing would be to use a conda-forge package.

Reply:
I've removed the file. Creating a specific tag/release would make sense, but since the current code is in an ongoing PR, I don't know if it's worth waiting on that one (I'd hope that upstream approves it and then releases a minor version)?